pyriemann/utils/mean.py

```diff
@@ -679,33 +674,29 @@ def mean_wasserstein(X, tol=10e-9, maxiter=50, init=None, sample_weight=None):
 
     References
     ----------
-    .. [1] `Geometric Radar Processing based on Frechet distance: Information
-        geometry versus Optimal Transport Theory
-        <https://ieeexplore.ieee.org/document/6042179>`_
-        F. Barbaresco. 12th International Radar Symposium (IRS), October 2011
+    .. [1] `Barycenter Estimation of Positive Semi-Definite Matrices with
+        Bures-Wasserstein Distance
+        <https://arxiv.org/abs/2302.14618>`_
+        J. Zheng, H. Huang, Y. Yi, Y. Li, S.-C. Lin.
```

Comment:
I would like us to add the reference to the seminal work of Agueh2011, in addition to the reference to the current implementation of Zheng2023.
```suggestion
    .. [1] `Barycenters in the Wasserstein space
        <https://hal.science/hal-00637399/file/AC_bary_revis.pdf>`_
        M. Agueh and G. Carlier. SIAM Journal on Mathematical Analysis, 2011
    .. [2] `Barycenter Estimation of Positive Semi-Definite Matrices with
        Bures-Wasserstein Distance
        <https://arxiv.org/abs/2302.14618>`_
        J. Zheng, H. Huang, Y. Yi, Y. Li, S.-C. Lin, ArXiv, 2023
```